### DISCUSSION TITLE
Discord link invalid

### DISCUSSION THREAD

**Discussion by ryansurf** [TreyWW/MyFinances#327] [General]
Hey everyone! The discord link is invalid... can someone update that or send me another one? Thanks

**Comment by TreyWW** [comment]
Hey @ryansurf, apologies for that.

The link on the [documentation](https://docs.strelix.org/MyFinances/) seems to still be valid, could you give that a shot?

Where was the link that you found originally?

**Reply by ryansurf** [reply]
Sorry @TreyWW, it was an issue on my end... I didn't have the discord app opened which lead to the invalid invite error. My apologies!

**Reply by TreyWW** [reply]
Cool, no problem! :)
